web/src/components/product/ProductSelectionProgress.jsx

```diff
@@ -35,15 +36,9 @@ import { PATHS } from "~/router";
  */
 function ProductSelectionProgress() {
   const { selectedProduct } = useProduct({ suspense: true });
-  const { manager } = useInstallerClient();
-  const [status, setStatus] = useState();
+  const { isBusy } = useInstallerStatus({ suspense: true });
 
-  useEffect(() => {
-    manager.getStatus().then(setStatus);
-    return manager.onStatusChange(setStatus);
-  }, [manager, setStatus]);
-
-  if (status === IDLE) return <Navigate to={PATHS.root} replace />;
+  if (!isBusy) return <Navigate to={PATHS.root} replace />;
```

Comment:
That was a good movement. I like to see how components are getting cleaner little by little.